agentic/src/nodes/diagnosticsIssueFix.ts

```diff
@@ -118,7 +118,7 @@ export class DiagnosticsIssueFix extends BaseNode {
         if (response.data.response?.tasks && response.data.response.yesNo) {
           nextState.shouldEnd = false;
           // group tasks by uris
-          const newTasks: { uri: string; tasks: string[] }[] =
+          let newTasks: { uri: string; tasks: string[] }[] =
```

Comment:
can we not filter out the list completely in the UI so agents don't even have to know about issues that weren't selected by the user?

Reply:
Moved the filter to the ui side. Passing back filtered tasks [here](https://github.com/konveyor/editor-extensions/pull/647/commits/a8dc28b0553f22bccb156de32606009758502e9e#diff-f3f5d841404c0744197395afe0f33a42d97632f263a367cd097368a0c344d8e0R58)